src/views/poi_bloc/Images.jsx

```diff
@@ -0,0 +1,55 @@
+/* global _ */
+import React from 'react';
+import classnames from 'classnames';
+
+function createImage(param, classes) {
+  return <a
+    target='_blank'
+    rel='noopener noreferrer'
+    href={ param.href }
+    className={ classes }
+    style={{ backgroundImage: param.url }}
+  />;
+}
+
+function createImages(images) {
+  if (images.length < 2) {
+    return;
+  } else if (images.length === 2) {
+    return <div className='poi_panel__pictures_block poi_panel__pictures_second_block'>
+      { createImage(images[1], 'poi_panel__pictures_block_full') }
+    </div>;
+  }
+  return <div
+    className={ classnames(
```

Comment:
No real need for `classnames` if you don't use its conditional class toggling capabilities.

Reply:
It allows to make a nicer display but I can remove it if you prefer.

Reply:
`Nicer display`? You mean code? This isn't important compared to importing smth we don't really use :)